source/portal/index.js

```diff
@@ -26,6 +26,7 @@ config.portal.ssl = config.portal.ssl || false;
 config.portal.force_tls_v12 = config.portal.force_tls_v12 || false;
 config.portal.reconnection_ticket_lifetime = config.portal.reconnection_ticket_lifetime || 600;
 config.portal.reconnection_timeout = Number.isInteger(config.portal.reconnection_timeout) ? config.portal.reconnection_timeout : 60;
+config.portal.cors = config.portal.cors || '*';
```

Comment:
'cors' should be a list if multiple origins are allowed.  E.g. 'localhost:3001', 'localhost:3004' are both allowed.